debian/copyright

```diff
@@ -0,0 +1,1278 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
If this file is necessary - could we generate it, not hard-push it and forget to update?

Reply:
Yes, this file can generated using some tools as highlighted [here](https://wiki.debian.org/CopyrightReviewTools). But that requires a bit more work than manually updating the file in my opinion. If you recommend, I can remove this file.

Reply:
copyright claims for `debian/*` scripts are contained in this file. Please don't remove it ......